src/settings-ui/Microsoft.PowerToys.Settings.UI/ViewModels/PowerRunPluginViewModel.cs

```diff
@@ -7,23 +7,22 @@
 using System.ComponentModel;
 using System.Linq;
 using System.Runtime.CompilerServices;
+using Microsoft.PowerToys.Settings.UI.Library;
 
-namespace Microsoft.PowerToys.Settings.UI.Library.ViewModels
+namespace Microsoft.PowerToys.Settings.UI.ViewModels
```

Comment:
why are we moving this file, unclear

Reply:
It's a part of PowerLauncherViewModel and is only used along with it.